website/docs/r/imp_app_template.html.markdown

```diff
@@ -46,9 +46,12 @@ The following arguments are supported:
 
 The config_list supports the following: 
 
-* `key` - (Optional) Configuration item key. Valid values: 
+* `key` - (Optional) Configuration item key. Valid values: ["config.appCallbackAuthKey","config.appCallbackUrl","config.callbackClass.live","config.callbackClass.user","config.livePullDomain","config.livePushDomain","config.multipleClientsLogin","config.regionId","config.streamChangeCallbackUrl"].
 * `value` - (Optional) Configuration item content.
 
+-> **NOTE:**  The `value` attribute of the `config_list` attribute returns empty by default. The `config_list` attribute has 9 keys, these 9 keys must all be set in the template, otherwise there will be a diff.
```

Comment:
改成这个：
By default, the attribute `config_list` will return all of nine keys with empty value. If you want to set one or more of the key's value, you had better also set other keys, otherwise, there will be a diff.

Reply:
done